src/providers/LayerProvider/LayerProvider.tsx

```diff
@@ -360,6 +366,7 @@ export const LayerProvider = ({
           read,
           syncTimestamps,
           readTimestamps,
+          expireDataCaches: resetCaches,
```

Comment:
This looks like a trivial change, but it's actually important to export this _in the `<LayerProvider>`_ as opposed to a standalone hook, otherwise we'll get two separate instances of the timestamps cache!